locations/IP.json

```diff
@@ -156,6 +161,10 @@
             },
             {
                 "name": "Big Key Chest",
+                "access_rules":[
+                    "@Ice Palace/Map Chest",
+                    "[@Ice Palace/Map Chest], somaria"

```

Comment:
here needs a check for glitched logic to be added. its needs to be at least $checkGlitches|1